Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -313,17 +371,383 @@ def zoom_list_user_messages(self, user_id: str, date_arg: datetime, from_arg: da
+    :param action_text: str: The text attached to the button, used for string replacement.
+    :param toJid: str: The Jid of where the question exists.
+    :param accountId: str: Zoom account ID
+    :param robotJid: str: Zoom BOT JID
+    :return: None
+    """
+    if '{user}' in entitlement_reply:
+        entitlement_reply = entitlement_reply.replace('{user}', str(user_name))
+    if '{response}' in entitlement_reply and action_text:
+        entitlement_reply = entitlement_reply.replace('{response}', str(action_text))
+
+    url_suffix = '/im/chat/messages'
+    content_json = {
+        "content": {
+            "body": [
+                {
+                    "type": "message",
+                    "text": entitlement_reply
+                }
+            ]
+        },
+        "to_jid": to_jid,
+        "robot_jid": robot_jid,
+        "account_id": account_id
+    }
+    await zoom_send_notification_async(CLIENT, url_suffix, content_json)
+
+
+async def answer_question(text: str, question: dict, email: str = ''):
+    entitlement = question.get('entitlement', '')
+    to_jid = question.get('to_jid')
+    guid, incident_id, task_id = extract_entitlement(entitlement)
+    try:
+        demisto.handleEntitlementForUser(incident_id, guid, email, text, task_id)
+        account_id = CLIENT.account_id
+        bot_jid = CLIENT.bot_jid
+        if account_id and bot_jid:
+            _ = await process_entitlement_reply(text, account_id, bot_jid, to_jid)
+    except Exception as e:
+        demisto.error(f'Failed handling entitlement {entitlement}: {str(e)}')
+    question['remove'] = True
+    return incident_id
+
+
+async def handle_text(investigation_id: str, text: str, operator_email: str, operator_name: str):
+    """
+    Handles text received from Zoom 
+
+    Args:
+        investigation_id: The mirrored investigation ID
+        text: The received text
+        operator_email: The sender email
+        operator_name: The sender name
+    """
+    if text:
+        demisto.addEntry(id=investigation_id,
+                         entry=text,
+                         username=operator_name,
+                         email=operator_email,
+                         footer=MESSAGE_FOOTER
+                         )
+
+
+async def handle_listen_error(error: str):
+    """
+    Logs an error and updates the module health accordingly.
+
+    Args:
+        error: The error string.
+    """
+    demisto.error(error)
+    demisto.updateModuleHealth(error)
+
+
+async def handle_mirroring(payload):
+    """
+    handle messages from the Zoom webhook that have been identified as possible mirrored messages
+    If we find one, we will update the mirror object and send
+    the message to the corresponding investigation's war room as an entry.
+    :param payload: str: The request payload from zoom
+    :return: None
+    """
+    channel_id = payload["object"].get("channel_id")
+    if not channel_id:
+        return
```

Comment:
I don't think it's necessary because it would make a lot of noise. 
This function will receive all the messages that are sent in zoom and potentialy came from a mirrored channel, and we want to throw away any message that is no longer a potential mirrored.